More GH Actions goodies

artembilan opened the pull request:
* Add RabbitMQ, MongoDB & Redis services into GH actions
* Remove Travis

artembilan committed:
More GH Actions goodies

* Add RabbitMQ, MongoDB & Redis services into GH actions
* Remove Travis

artembilan committed:
* Add `distributionSha256Sum` into `gradle-wrapper.properties`
* Try `TERM: dumb` to see difference for Gradle output in GH action
* Add `-S` to see the reason of test failure

artembilan committed:
* Try MQTT plugin for RabbitMQ image

garyrussell commented on .github/workflows/pr-build-workflow.yml:
```diff
@@ -10,12 +10,30 @@ jobs:
       rabbitmq:
         image: rabbitmq
         ports:
           - 5672:5672
```
`- 15762:15762` for management plugin?

garyrussell replied:
Forget that; not used in this project; only spring-rabbit.

artembilan committed:
* Try `-i` for test failure in Gradle on Actions

artembilan committed:
* Use `eclipse-mosquitto` image for MQTT on actions
* Fix `FtpServerOutboundTests` for the proper FTP file list
* Output file list for the failed session in the `RotatingServersTests`

artembilan committed:
* Adjust SOUT in the test for Checkstyle

artembilan committed:
* Make some change to `RotatingServersTests`.
Looks like the order for `@BeforeAll` methods is not determined
and we may start to interaction with FTP server which is not
started yet

garyrussell committed:
Fix SSL Handshake Test

- we already made this change for the NIO test, but the NET test is failing the same
  way - the cert failure switched to the server so the expected exception was not
  thrown on the client side. Due to JVM changes.

artembilan committed:
* Fix SFTP tests for wrong host and auto-startup state
* Remove `-i` for Gradle in the GH Actions

artembilan committed:
* Try `--no-parallel` for Gradle in the GH Actions

artembilan committed:
* Try GH actions without Mosquitto

artembilan committed:
* Try `cyrilix/rabbitmq-mqtt` for GH Actions

artembilan commented:
It doesn't show the time for each module to pass its tests, but I think we should believe it that it works well.
And therefore it is ready for merging.

Thanks
